### PR TITLE
Fix #667 Session1 scheduling overview

### LINKED ISSUE #667: 第一部分 4.1 调度概述
请按照 [审核标准](https://github.com/pingcap-incubator/tidb-in-action/blob/master/README.md) 审核以下章节：
第一部分 [4.1 调度概述](https://github.com/pingcap-incubator/tidb-in-action/tree/master/session1/chapter4/scheduling-overview.md)

### PR DESCRIPTION
**Pull request by wulinning133**
### What problem does this PR solve? 
This PR is try to fix [#667](https://github.com/pingcap-incubator/tidb-in-action/issues/667)

### How to contact you? 
- Email:
- Telephone:
- WeChat: wulinning
- Others:

### CODE REVIEW

File: session1/chapter4/scheduling-overview.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ TiKV 集群是 TiDB 数据库的分布式 KV 存储引擎，数据以 Region 为
 * 如何保证同一个 Region 的多个 Replica 分布在不同的节点上？更进一步，如果在一台机器上启动多个 TiKV 实例，会有什么问题？
 * TiKV 集群进行跨机房部署的时候，如何保证一个机房掉线，不会丢失 Raft Group 的多个 Replica？
 * 添加一个节点进入 TiKV 集群之后，如何将集群中其他节点上的数据搬过来?
-* 当一个节点掉线时，会出现什么问题？整个集群需要做什么事情？如果节点只是短暂掉线（重启服务），那么如何处理？如果节点是长时间掉线（磁盘故障，数据全部丢失），需要如何处理？假设集群需要每个 Raft Group 有 N 个副本，那么对于单个 Raft Group 来说，Replica 数量可能会不够多（例如：节点掉线，失去副本），也可能会过于多（例如：掉线的节点又恢复正常，自动加入集群）。那么如何调节 Replica 个数？
+* 当一个节点掉线时，会出现什么问题？整个集群需要做什么事情？如果节点只是短暂掉线（重启服务），那么如何处理？如果节点是长时间掉线（磁盘故障，数据全部丢失），需要如何处理？假设集群需要每个 Raft Group 有 N 个副本，那么对于单个 Raft Group 来说，Replica 数量可能会不够（例如节点掉线，失去副本），也可能会过于多（例如掉线的节点又恢复正常，自动加入集群）。那么如何调节 Replica 个数？
```
Comment (AndreMouche) [inline session1/chapter4/scheduling-overview.md]: 把这个问题归类，是否会更清晰一些呢？
```suggestion
* 当一个节点掉线时，会出现什么问题？整个集群需要做什么事情？
	* 从节点的恢复时间来看
    	* 如果节点只是短暂掉线（重启服务），如何处理？
    	* 如果节点是长时间掉线（磁盘故障，数据全部丢失），如何处理？
	* 假设集群需要每个 Raft Group 有 N 个副本，从单个 Raft Group 的 Replica 个数来看
		* Replica 数量不够（例如节点掉线，失去副本），如何处理？
		* Replica 数量过多（例如掉线的节点又恢复正常，自动加入集群），如何处理？
```

File: session1/chapter4/scheduling-overview.md [hunk 2]
```diff
@@ -63,9 +63,9 @@ TiKV 节点（Store）与 PD 之间存在心跳包，一方面 PD 通过心跳
 * 数据写入/读取速度
 * 发送/接受的 Snapshot 数量（Replica 之间可能会通过 Snapshot 同步数据）
 * 是否过载
-* labels标签信息（标签是具备层级关系的一系列 Tag）
+* labels 标签信息（标签是具备层级关系的一系列 Tag）
 
-**每个 Raft Group 的 Leader 会定期向 PD 汇报信息。**
+- **每个 Raft Group 的 Leader 会定期向 PD 汇报 Region 的状态信息**
```
Comment (AndreMouche) [inline session1/chapter4/scheduling-overview.md]:
```suggestion
**每个 Raft Group 的 Leader 会定期向 PD 汇报 Region 的状态信息**
```
看起来前面的这个 `-` 是误操作？排版好像不太对了

File: session1/chapter4/scheduling-overview.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ TiKV 集群是 TiDB 数据库的分布式 KV 存储引擎，数据以 Region 为
 ### 4.1.4 信息收集
 调度依赖于整个集群信息的收集，简单来说，我们需要知道每个 TiKV 节点的状态以及每个 Region 的状态。TiKV 集群会向 PD 汇报两类消息：
 
-**每个 TiKV 节点会定期向 PD 汇报节点的整体信息**
+- **每个 TiKV 节点会定期向 PD 汇报节点的状态信息**
```
Comment (AndreMouche) [inline session1/chapter4/scheduling-overview.md]:
```suggestion
**每个 TiKV 节点会定期向 PD 汇报节点的状态信息**
```
看起来前面的这个 - 是误操作？排版好像不太对了

**Review by AndreMouche — COMMENTED** [review]
Rest LGTM

**Comment by wulinning133** [comment]
ACK

**Review by kissmydb — APPROVED** [review]
LGTM

**Pull request merged**